pages/profile/index.js

```diff
@@ -1,5 +1,78 @@
+import useSWR from "swr";
+import { useState } from "react";
+import postImage from "@/lib/postImage";
+import patchData from "@/lib/patchData";
 import User from "@/components/User/User";
+import createProfileDbData from "@/lib/createProfileDbData";
+import {
+  EditIcon,
+  CancelIcon,
+  EditButton,
+} from "@/components/Details/StyledDetails";
+import ProfileForm from "@/components/ProfileForm/ProfileForm";
 
-export default function Routes() {
-  return <User />;
+export default function Profile() {
+  const {
+    data: profiles,
+    isLoading,
+    error,
+    mutate: refetchProfiles,
+  } = useSWR("/api/users");
+  const [edit, setEdit] = useState(false);
+  const [image, setImage] = useState(null);
+  const [sendingForm, setSendingForm] = useState(false);
```

Comment:
sticking to the naming convention would make it clear to all other developers in all instances where you use the state and the setter function that they are working with a boolean: 

```suggestion
  const [isSending, setIsSending] = useState(false);
```